Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -1676,6 +1677,17 @@ def prepare_PAT(self, sequence = "miniAOD"):
 
         return
 
+    def prepare_PATGEN(self, sequence = "miniAOD"):
+        ''' Enrich the schedule with PATGEN '''
+        self.loadDefaultOrSpecifiedCFF(sequence,self.PATDefaultCFF,1) #this is unscheduled
```

Comment:
actually, reconciling the comments above,
it would be better to remove PATGEN and put all conditionals in step PAT for a sequence "miniGEN".